src/DynamoCoreWpf/UI/Prompts/CrashPrompt.xaml.cs

```diff
@@ -69,6 +79,34 @@ public CrashPrompt(CrashPromptArgs args, DynamoViewModel dynamoViewModel)
                 ConvertFormattedTextIntoTextblock(this.txtOverridingText, overridingText);
             }
         }
+        private string PackagesNamesToMakrdown(IEnumerable<string> packagesNames)
+        {
+            string markdownText = "";
+            if (packagesNames != null)
+            {
+                markdownText = "## Loaded Packages" + Environment.NewLine;
+                foreach (var name in packagesNames)
+                {
+                    markdownText += "- " + name + Environment.NewLine;
+                }
+            }
+            return markdownText;
+        }
+
+        private IEnumerable<string> PackagesNames(PackageLoader packageLoader)
+        {
+            var packages = packageLoader.LocalPackages;
+            var pckgsNames = new List<string>();
+            if (packages.Any())
+            {
+                pckgsNames.AddRange(packages.Select(pckg => pckg.Name));
+            }
+            else
+            {
+                pckgsNames.Add("No packages where loaded.");
+            }
+            return pckgsNames;
+        }
```

Comment:
I would maybe consider moving these functions to https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Utilities/CrashUtilities.cs